src/style.css

```diff
@@ -0,0 +1,105 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 900;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+input,
+input::placeholder {
+  font-family: inherit;
+}
+
+.inputWrapper {
+  position: relative;
+  display: block;
+}
+
+.inputWrapperBig {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.inputWrapper::before {
+  content: "";
+
+  display: block;
+  width: 19px;
+  height: 19px;
+
+  position: absolute;
+  top: 25px;
+  left: 26px;
+
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
```

Comment:
combine in `background` property

Reply:
Please be so kind to claify which stylies can be combined